### PR TITLE
Chore/add intro paragraph to docs.claudie.io

### PR DESCRIPTION
**Pull request by JKBGIT1**
closes https://github.com/berops/claudie/issues/1244

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,56 +1,35 @@
-# Docs Overview
+# What is Claudie
 
-In case you are reading our documentation in our [GitHub repository](https://github.com/berops/claudie/), you might want to read the prettier version of it on [docs.claudie.io](https://docs.claudie.io/)
+Claudie is a platform for managing multi-cloud and hybrid-cloud Kubernetes clusters with support for nodepools across different cloud-providers and on-premise data centers.
```
Comment (bernardhalas) [inline docs/index.md:3]:
```suggestion
Claudie is a platform for managing multi-cloud and hybrid-cloud Kubernetes clusters. These Kubernetes clusters can mix and match nodepools from various cloud providers, e.g. a single cluster can have a nodepool in AWS, another in GCP and another one on-premises. This is our opinionated way to build multi-cloud and hybrid-cloud Kubernetes infrastructure. On top of that Claudie supports Cluster Autoscaler on the managed clusters.
```

**Review by bernardhalas — CHANGES_REQUESTED** [review]
I've left minor comments for your assessment. Thank you @JKBGIT1

**Comment by JKBGIT1** [comment]
Hi @bernardhalas please take a look at the changes.

**Review by bernardhalas — APPROVED** [review]
Thanks @JKBGIT1

**Pull request merged**